x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/hot_phase/hot_phase.tsx

```diff
@@ -107,7 +108,7 @@ export const HotPhase: FunctionComponent = () => {
                   <EuiSpacer size="s" />
                   <FormattedMessage
                     id="xpack.indexLifecycleMgmt.editPolicy.hotPhase.rolloverDefaultsTipContent"
-                    defaultMessage="Rollover when an index is 30 days old or reaches 50 gigabytes."
+                    defaultMessage="Rollover when an index is 30 days old or the primary shard reaches 50 gigabytes."
```

Comment:
An index can have multiple primary shards. 
```suggestion
                    defaultMessage="Roll over when an index is 30 days old or any primary shard reaches 50 gigabytes."
```